src/main/java/micdoodle8/mods/galacticraft/core/nei/NEIGalacticraftConfig.java

```diff
@@ -48,6 +49,13 @@ public void loadConfig() {
             }
         }
 
+        if (Loader.isModLoaded(Constants.MOD_ID_NEW_HORIZONS_CORE_MOD)) {
+            API.hideItem(new ItemStack(GCItems.basicItem, 1, 3));
+            API.hideItem(new ItemStack(GCItems.basicItem, 1, 4));
+            API.hideItem(new ItemStack(GCItems.basicItem, 1, 5));
+            API.hideItem(new ItemStack(GCItems.meteoricIronIngot, 1, 0));
+        }
+
```

Comment:
Is this needed if the `GCCoreUtil.registerGalacticraftItem()` calls never occur?

Reply:
yes.
that method does not really do much. gameregistry is added by `GCItems.registerItem(GCItems.basicItem);` if that is what you were expecting here.